source/administration/legacy-upgrade.md

```diff
@@ -1,17 +1,109 @@
+          - **Team Edition**: `wget https://releases.mattermost.com/X.X.X/mattermost-team-X.X.X-linux-amd64.tar.gz`
+      2. Review past and upcoming deprecation notices [listed on our website](https://about.mattermost.com/deprecated-features/).
+2. Stop the Mattermost Server
+      1. Consider posting an announcement to active teams about stopping the Mattermost server for an upgrade.
+      2. To stop the server run `sudo stop mattermost`.
+3. Backup your data
+      1. Back up your `config.json` file, which contains your system configuration. This will be used to restore your current settings after the new version is installed.
+      2. Backup your database using your organization's standard procedures for backing up MySQL or PostgreSQL.
+      3. If you're using local file storage, back up the location where files are stored.
+4. Install new version
+      1. Run `tar -xvzf mattermost-team-X.X.X-linux-amd64.tar.gz` to decompress the upgraded version and replace the current version of Mattermost on disk, where `X.X.X` is the version number to which you are upgrading.  
+5. Restore the state of your server
+      1. Copy the backed up version of `config.json` in place of the default `config.json`.
+6. Start your server and address any setting changes relevant in the latest version of Mattermost
+      1. Run `sudo start mattermost`.
+      2. Opening the **System Console** and saving a change will upgrade your `config.json` schema to the latest version using default values for any new settings added.
+7. If you have TLS set up on your Mattermost server, you must activate the CAP_NET_BIND_SERVICE capability to allow the new Mattermost binary to bind to low ports.
+
+      1. `cd {install-path}`
+      2. `sudo setcap CAP_NET_BIND_SERVICE=+ep ./bin/platform`
+
+After the server is upgraded, users might need to refresh their browsers to experience any new features.
+
+## Upgrade Enterprise Edition from 3.0.0 and later
+
+### Important Notices
+
+1. Security related changes were made in 3.9.0 that cause any previously created team invite links, password reset links, and email verification links to no longer work. You must update any place where you have published these links.
+
+2. Security-related changes were made in 3.8.0 that require you to verify settings in the System Console before upgrading from version 3.7.0 and earlier to any version greater than 3.8.0
+
+  1. In the GENERAL section of the System Console, click **Configuration** and make sure that the **Site URL** is specified. It must not be empty. For more information about SiteURL, see `Configuration Settings <config-settings.html#site-url>`_
+  2. In the GENERAL section of the System Console, click **Logging** and make sure that the **File Log Directory** field is either empty or has a directory path only. It must not have a filename as part of the path.
+
+3. Changes were made in 3.8.0 that require a change in the proxy configuration. If you're using NGINX:
```

Comment:
These appear as a single bullet list for me

![image](https://cloud.githubusercontent.com/assets/13119842/26068458/9fece5a0-396b-11e7-8be4-caaf00296271.png)